recipe/meta.yaml

```diff
@@ -32,6 +32,7 @@ requirements:
     - requests-kerberos
     - tqdm >=4.0.0
     - pandas >=0.23.0
+    - python-highcharts
```

Comment:
No, it's not needed. I was exploring the error messages in different setting. :)
It currently fails because it cannot find vertica_highchart on Conda. I'm planning to publish vertica_highchart on conda-forge in hope that it will resolve our problem here.